nmxact/xact/image.go

```diff
@@ -120,13 +136,16 @@ func findChunkLen(s sesn.Sesn, hash []byte, upgrade bool, data []byte,
 			return 0, err
 		}
 
-		if len(enc) <= s.MtuOut() {
+		if len(enc) <= (s.MtuOut() - buffer) {
```

Comment:
Not a big deal, but the parentheses are unnecessary here.

Reply:
done